### PR TITLE
Remove samples

### PR DESCRIPTION
**Pull request by 2m**
And migrate sample code that was used in docs to the docs or other projects.

Part of #21639

### CODE REVIEW

**Comment by 2m** [comment]
OK TO TEST

**Comment by ktoso** [comment]
Touch to review, LGTM I think.

**Review by ktoso — APPROVED** [review]

**Review by drewhk — APPROVED** [review]

File: akka-docs/rst/java/code/docs/cluster/MetricsListener.java [hunk 1]
```diff
@@ -33,21 +33,23 @@ public void postStop() {
...
   @Override
-  public Receive createReceive() {
-    return receiveBuilder()
-      .match(ClusterMetricsChanged.class, clusterMetrics -> {
-        for (NodeMetrics nodeMetrics : clusterMetrics.getNodeMetrics()) {
-          if (nodeMetrics.address().equals(cluster.selfAddress())) {
-            logHeap(nodeMetrics);
-            logCpu(nodeMetrics);
-          }
+  public void onReceive(Object message) {
```
Comment (johanandren) [inline akka-docs/rst/java/code/docs/cluster/MetricsListener.java]: We should update this to the new Java api as well. I'll open a ticket.

Comment (johanandren) [inline akka-docs/rst/java/code/docs/cluster/MetricsListener.java]: Added #22308

**Review by johanandren — APPROVED** [review]
LGTM!

**Pull request merged**